sample-lnd.conf

```diff
@@ -1401,6 +1406,9 @@
 ; Example:
 ;   db.postgres.maxconnections=
 
+; Whether to skip executing schema migrations.
+; db.postgres.skipmigrations=false
```

Comment:
when would we want to set skip migrations = true from the config? Is it only for testing?